kubejs/server_scripts/base/recipes/minecraft/shapeless.js

```diff
@@ -0,0 +1,3 @@
+ServerEvents.recipes(event => {
+    event.shapeless('minecraft:crafting_table', ['#forge:workbench']);
```

Comment:
perhaps add `const recipes`,, `id_prefix` and the whole shebang?

Reply:
Good thinking :D